homeassistant/components/alexa/entities.py

```diff
@@ -791,19 +791,18 @@ def interfaces(self):
         yield Alexa(self.hass)
 
     def _check_requirements(self):
-        """Check the hass URL for HTTPS scheme and port 443."""
+        """Check the hass URL for HTTPS scheme."""
         if "stream" not in self.hass.config.components:
-            _LOGGER.error(
+            _LOGGER.debug(
                 "%s requires stream component for AlexaCameraStreamController",
                 self.entity_id,
             )
             return False
 
         url = urlparse(network.async_get_external_url(self.hass))
-        if url.scheme != "https" or (url.port is not None and url.port != 443):
```

Comment:
I think that this check is still necessary. This user set their base url to a url that uses port 443: https://github.com/home-assistant/core/issues/34030#issuecomment-616510198

Reply:
The original reason to check for this was because the echo show devices I tested with required the stream use HTTPS, and connected only on port 443. In typical situations this connection occurs within the users LAN. And since users might not use HTTPS on port 443 inside their LAN, I was trying to eliminate issue reports with the check.

I removed the check for the following reasons:
-  Based on another user report #34437 the FireTV stick works with a non-443 HTTPS port connection. 
-  I also encountered a user in the forums that uses port forwarding at the router with hairpin NAT, this check prevented them from using this feature even though his environment would meet requirements for the echo device.
-  Same port forward can be accomplished with the reverse proxy addon.
-  There are also 3rd party "Alexa bult-in" devices that use the Alexa Voice Service and can choose to handlle the stream URLs on any port.

Reply:
@balloob I asked the user to provide more details. See the user's follow up https://github.com/home-assistant/core/issues/34030#issuecomment-616911974 regarding configuration.

Reply:
The way I read #34437 is that HAASKA works on a port that's not 443 (which makes sense, it's custom code). 

But sure, let's drop it and see what happens.